reporter/src/main/kotlin/reporters/GitlabLicenseModelReporter.kt

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.reporter.reporters
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.dataformat.yaml.YAMLMapper
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+
+import java.io.File
+
+import org.ossreviewtoolkit.reporter.Reporter
+import org.ossreviewtoolkit.reporter.ReporterInput
+import org.ossreviewtoolkit.reporter.gitlab.LicenseModelMapper
+
+/**
+ * Creates YAML documents according to the Gitlab license model schema version 2.1, see
+ * https://gitlab.com/gitlab-org/security-products/license-management/-/blob/master/spec/fixtures/schema/v2.1.json.
+ * Examples can be found under
+ * https://gitlab.com/gitlab-org/security-products/license-management/-/tree/master/spec/fixtures/expected.
+ *
+ * The option key [SKIP_EXCLUDED] specifies whether to omit excluded packages. Valid values are 'true' and 'false' and
+ * the default is 'false'.
+ */
+class GitlabLicenseModelReporter : Reporter {
+    companion object {
+        const val SKIP_EXCLUDED = "skip.excluded"
```

Comment:
Nit: This could be a private top-level variable to get rid of the companion object.

Reply:
I prefer to keep it like this as I use it from the test in the next commit.

Reply:
My goal was to get rid of the companion object, not to make the variable private. But if you also don't want to make the variable `internal` (and probably give it a prefix when moved out of the class, too), I'm fine with keeping it here.

Reply:
I'll keep it then.